js/valid.js

```diff
@@ -0,0 +1,15 @@
+//=======Form Validation=========
+const contactForm = document.getElementById('contact-form');
+const email = document.querySelector('#email');
+const errorDisplay = document.querySelector('#errorDisplay');
+const checker = /^[a-z0-9._-]+@[a-z0-9.-]+\.[a-z]{2,4}$/g;
+contactForm.addEventListener('submit', (e) => {
+  if (!email.value.match(checker)) {
+    e.preventDefault();
+    errorDisplay.style.visibility = 'visible';
+    errorDisplay.classList.add('error-message');
+    errorDisplay.textContent = '*Your email address should be all in lowercase';
```

Comment:
- [ ] Great job with the email validation. According to the guidelines, the error message should pop up around the submit button. This is to help the user see it easily. Kindly fix that. 😄 
![Screenshot (274)_LI](https://user-images.githubusercontent.com/65298048/181996835-72fbfdbd-065e-4f97-89d0-19fc21e6cced.jpg)